e2e/test/scenarios/documents/card-embed-node.cy.spec.ts

```diff
@@ -22,7 +22,7 @@ describe("documents card embed node custom logic", () => {
         idAlias: "documentId",
       });
 
-      cy.get("@documentId").then((id) => cy.visit(`/document/${id}`));
+      H.visitDocument("@documentId");
```

Comment:
The PR also uses `H.visitDocument` where possible - it has an additional `cy.wait` that should help with general flakiness.